egs/sprakbanken_swe/s5/run.sh

```diff
@@ -0,0 +1,164 @@
+cd #!/bin/bash
+
+. ./cmd.sh ## You'll want to change cmd.sh to something that will work on your system.
+           ## This relates to the queue.
+. ./path.sh # so python3 is on the path if not on the system (we made a link to utils/).a
+
+# This is a shell script, but it's recommended that you run the commands one by
+# one by copying and pasting into the shell.
+
+
+# Download the corpus and prepare parallel lists of sound files and text files
+# Divide the corpus into train, dev and test sets
+local/sprak_data_prep.sh  || exit 1;
+utils/fix_data_dir.sh data/train
+
+# Perform text normalisation, prepare dict folder and LM data transcriptions
+# this will download the lexicon directly from sprakbanken, without changes made by the author on the original lexicon
+# local/dict_prep.sh
+local/copy_dict
+
+utils/prepare_lang.sh data/local/dict "<UNK>" data/local/lang_tmp data/lang || exit 1;
+
+# Now make MFCC features.
+# mfccdir should be some place with a largish disk where you
+# want to store MFCC features.
+mfccdir=mfcctest
+
+
+# Extract mfccs 
+# p was added to the rspecifier (scp,p:$logdir/wav.JOB.scp) in make_mfcc.sh because some 
+# wave files are corrupt 
+# Will return a warning message because of the corrupt audio files, but compute them anyway
+# If this step fails and prints a partial diff, rerun from sprak_data_prep.sh
+
+steps/make_mfcc.sh --nj 10 --cmd $train_cmd data/test exp/make_mfcc/test test mfcc
+steps/make_mfcc.sh --nj 10 --cmd $train_cmd data/train exp/make_mfcc/train mfcc || exit 1;
```

Comment:
An example of the issue mentioned above -- both make_mfcc.sh calls should be ending with 
```
... || exit 1
```